app/scripts/lib/app-start.js

```diff
@@ -60,6 +60,7 @@ define([
   'models/auth_brokers/iframe',
   'models/unique-user-id',
   'models/user',
+  'models/verification/same-browser',
```

Comment:
Eh we can stay with "Same browser", just as long everyone is on board.

other ideas: "original browser", "existing browser"